desktop-widgets/preferences/preferences_defaults.cpp

```diff
@@ -21,6 +22,12 @@ PreferencesDefaults::~PreferencesDefaults()
 	delete ui;
 }
 
+/*void PreferencesDefaults::on_resetSettings_clicked()
+{
+	auto dialog = PreferencesDialog::instance();
+	dialog->defaultsRequested();
+} */
```

Comment:
more code added that is commented out... that's just not a good way to do things...